package.json

```diff
@@ -384,7 +384,7 @@
     "chromedriver": "^121.0.1",
     "classnames": "^2.3.1",
     "compare-versions": "3.5.1",
-    "cypress": "9.5.4",
+    "cypress": "12.17.4",
```

Comment:
plugins if they dont run `yarn osd bootstrap --single-version=loose` might run into issues if they define the cypress version that is isn't this one

Reply:
something to consider mimicking 

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.x/package.json#L170 `opensearch-next`

Reply:
but i wouldn't block this pr since its going to feature branch but might need it for main

Reply:
build repo defines cypress version 12.13 https://github.com/opensearch-project/opensearch-build/blob/0931ccc2752d9eced6cb3015b822b4a10742fc80/docker/ci/dockerfiles/current/test.almalinux8.systemd-base.x64.arm64.ppc64le.dockerfile#L58

we might need to create an issue to bump the version in the build repo

Reply:
do we need to update this ? https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/package.json#L59

Reply:
Makes sense. Let me take a look at all the instances of cypress in our runners and other repos and I'll put together a small overview of what's needed to get all of these on a unified version of cypress.